src/Spice86.Core/Emulator/InterruptHandlers/Dos/Ems/EmmMapping.cs

```diff
@@ -0,0 +1,32 @@
+namespace Spice86.Core.Emulator.InterruptHandlers.Dos.Ems; 
+
+public record EmmMapping {
+    public const int Size = 4;
+    private byte[] _data { get; init; } = new byte[4];
+    
+    public ushort Handle {
+        get => (ushort) ((_data[0] & 0xFF) | ((_data[1] & 0xFF) << 8));
+        set {
+            _data[0] = (byte) (value & 0xFF);
```

Comment:
Accesses look like what the setWord method would do on the memory bus. Maybe there is something we could do to put this kind of code in common with indexers?